ERCS/erc-7786.md

```diff
@@ -109,6 +116,14 @@ After a sender has invoked `sendMessage`, further action MAY be required by the
 
 The exact interface for any such action is out of scope of this ERC. If the `postProcessingOwner` attribute is supported and present, such actions MUST be restricted to the specified account, otherwise they MUST be able to be performed by any party in a way that MUST NOT be able to compromise the eventual receipt of the message.
 
+#### `requestRelay`
+
+Requests that the message identified by `outboxId` be delivered to the receiver (i.e., relayed).
+
+Pre-pays via `msg.value` for the execution of the message on the receiver up to `gasLimit`. A `relayerFee` should be deducted from this to compensate the relayer.
```

Comment:
what if the destination chain has a different custom base token?

Reply:
This is another open question for me. It could have to be paid directly in the destination chain gas token, but this may not be the UX we want. It may need to be swapped for the destination chain token, either at source or destination, though this sounds like it may leave the sender vulnerable to attacks if not designed correctly.